visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -338,7 +349,11 @@ void MotionPlanningFrame::configureWorkspace()
 
 void MotionPlanningFrame::configureForPlanning()
 {
```

Comment:
I'm not convinced of that approach. Changing the start state to the current state just before planning, essentially will modify a manually adjusted start state. Consider the following work flow: 
- adjust start state to current to sync with current pose
- then manually adjust the start state to some other pose
- plan for this motion

I think, execution of motion should be disabled if start state deviates too strongly from current state.